src/docs/sphinx/Publications.rst

```diff
@@ -2,42 +2,40 @@
 Publications
 ###############################################################################
 
-Last updated 14-July-2020
+Last updated 22-January-2021
 
-Preprints
-=========
+
+2020
+====
 
 .. list-table::
    :widths: 100
    :header-rows: 0
 
    * - | **Simulation of coupled multiphase flow and geomechanics in porous media with embedded discrete fractures**
        | M Cusini, JA White, N Castelletto, RR Settgast
-       | arXiv preprint
-       | `arXiv:2007.05069 <https://arxiv.org/abs/2007.05069>`_
+       | International Journal for Numerical and Analytical Methods in Geomechanics (2020)
+       | `doi:10.1002/nag.3168 <https://doi.org/10.1002/nag.3168>`_
```

Comment:
This paper (and others) is not actually assigned to a volume. I think it'd be easier to keep all papers not published in a separate block, e.g. `Preprint / Accepted` until they are published.

Reply:
That will double the amount of work we have to do updating papers.  How about we delete all volume information, and just put the journal title?  Once it has a DOI link, that is all that is needed.  It will always be up to date.

Reply:
I think it is actually simpler. When we update the publication list (twice a year?) we populate with the most up-to-date information. If assigned to an issue/volume, we move it to the appropriate year.

I agree that the DOI link is ideal, but it does not solve the year issue. For example _Camargo et al., COMG (2020)_ has a DOI, but it in early view. It will be (hopefully) published in 2021.